src/yachtzee/fishy.ts

```diff
@@ -162,6 +162,37 @@ export function optimizeForFishy(yachtzeeTurns: number, setup?: boolean): number
         }
       },
     },
+    {
+      name: "Pocket Wish",
+      turns: 20 + (haveFishyPipe ? 10 : 0),
+      cost: mallPrice($item`pocket wish`),
+      action: () => {
+        acquire(1, $item`pocket wish`, 1.2 * mallPrice($item`pocket wish`));
+        if (!have($item`pocket wish`)) {
+          throw new Error("Unable to obtain Pocket Wish");
+        }
+        cliExecute(`genie effect Fishy`);
+        if (haveFishyPipe && haveEffect($effect`Fishy`) + adventureExtensionBonus < yachtzeeTurns) {
+          use(1, $item`fishy pipe`);
+        }
+      },
+    },
+    {
+      name: "2x Pocket Wish",
+      turns: 40 + (haveFishyPipe ? 10 : 0),
```

Comment:
We should probably stop adding this separately to every fishy source (out of scope of this PR)